collectors/ciscoamp/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ciscoamp-collector",
-  "version": "1.0.24",
+  "version": "1.0.26",
```

Comment:
why not `1.0.25`?

Reply:
Yes! I saw the Fix changes, Due Security Issues I was unable to check anything from last 2 Days. So I haven’t got chance to raise PR.